docs/data/catalog/platform/gc-forms/templates.md

```diff
@@ -0,0 +1,104 @@
+# Platform / GC Forms / Templates
+
+### :warning: Note
+This dataset is still in testing and only a snapshot of Staging data is available in Superset. 
+
+---
+
+Dataset providing GC Forms template data.  There are 3 tables as part of this dataset:
+
+- `template`: the templates that are used by GC Forms to render the form users fill out and submit.
+- `templateToUser`: a mapping of templates to the users that created them.
+- `user`: the users that have created a template in GC Forms.
```

Comment:
Basically it's a list of anyone who's ever logged in.  They may have created forms in the past but if they marked them for deletion there wont' be a corresponding entry in the `template` table.